Makefile

```diff
@@ -21,7 +21,7 @@ host_cleanup:
 	./host_cleanup.sh
 
 test:
-	./tests/05_test.sh
+	./tests/test.sh
```

Comment:
but `test.sh` does not exist under the `tests` folder: https://github.com/metal3-io/metal3-dev-env/tree/main/tests

Reply:
yes I changing `05_test.sh` that exists to `test.sh` I mean just renaming a file

Reply:
I am not sure if I understand this

Reply:
ah I overlooked file renaming change completely, sorry